c/examples/haploid_wright_fisher.c

```diff
@@ -41,10 +40,10 @@ simulate(
             child = tsk_node_table_add_row(
                 &tables->nodes, 0, t, TSK_NULL, TSK_NULL, NULL, 0);
             check_tsk_error(child);
-            left_parent = parents[gsl_rng_uniform_int(rng, N)];
-            right_parent = parents[gsl_rng_uniform_int(rng, N)];
+            left_parent = parents[(size_t)((rand()/(1.+RAND_MAX))*N)];
```

Comment:
You mentioned putting in a comment here to explain why it's not a great idea to do this. Think it would be a good idea!

Reply:
Yeah, I added it to the `rst`file in a recent commit.